cmd/main.go

```diff
@@ -130,29 +130,26 @@ func StartIPPoolController(mgr ctrl.Manager, commonService common.Service) {
 	}
 }
 
-func StartVPCController(mgr ctrl.Manager, commonService common.Service) {
+func StartVPCController(mgr ctrl.Manager, commonService common.Service, vpcService *vpc.VPCService) {
```

Comment:
Do we still need common.Service?

Reply:
+1, could remove it in both StartVPCController StartNamespaceController